test/index.js

```diff
@@ -3,6 +3,10 @@ var compatNode = require('../')
 var compatBrowser = require('../browser')
 var fixtures = require('./fixtures')
 
+// put NaN and Infinity
+fixtures.invalid[fixtures.invalid.length - 2].dkLen = NaN
+fixtures.invalid[fixtures.invalid.length - 1].dkLen = Infinity
```

Comment:
I'd rather add these fixtures inline here,  then do this weird modification which is error prone for others adding tests.